package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.ts",
   "scripts": {
     "lint": "npx eslint . --fix",
-    "start": "NODE_EXTRA_CA_CERTS=./db-ssl-certificate.pem ts-node src/index.ts",
+    "start": "NODE_EXTRA_CA_CERTS=./db-ssl-certificate.pem ts-node src/index.ts 2>&1 | tee /tmp/debug",
```

Comment:
we use yarn start in production too, with extra command stuff that logs to ./log and is already hooked into logging, so let's use a different command for this or remove it? we can potentially improve this later

Reply:
moved to start-logs